testing/features/support/env.rb

```diff
@@ -24,4 +24,4 @@
 CaTesting::Patches::Capybara.new.patch!
 CaTesting::Patches::SeleniumLogger.new.patch!
 CaTesting::Patches::SeleniumManager.new.patch!
-CaTesting::Patches::SeleniumOptions.new(ENV["BROWSER"].to_sym).patch!
+CaTesting::Patches::SeleniumOptions.new(ENV["BROWSER"]).patch!
```

Comment:
Just FYI you are never setting an env value called `BROWSER` in the saucelabs-workflow. There is one called `BROWSER_NAME` though?

Reply:
true! the way Saucelabs name some capabilities are different from BrowserStack thanks David.